django/db/models/sql/query.py

```diff
@@ -411,12 +411,12 @@ def get_aggregation(self, using, added_aggregate_names):
         # aren't smart enough to remove the existing annotations from the
         # query, so those would force us to use GROUP BY.
         #
-        # If the query has limit or distinct, then those operations must be
-        # done in a subquery so that we are aggregating on the limit and/or
+        # If the query has limit, distinct or uses set operations, then those operations
```

Comment:
`# If the query has limit or distinct, or uses set operations,` (also wrap the comment at 79 chars)